src/python/pants/backend/python/subsystems/test_pex_build_util.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import os
+import pathlib
```

Comment:
Thank you for using this library! Much nicer to work with.

Reply:
In this case, mixed with `os.walk` it's actually worse since that stdlib is pathlib accepting but not pathlib returning; thus awkward bits like `dir_path = pathlib.Path(root) / d`. That said, this should be nicer some day. The only thing better really is the possibility of type-checking Path vs str. That, though, is also a long way off. I tried to turn on type checking for the test target and the tangle is too deep. That will require concerted effort.